web_environment_ribbon/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,8 @@
+* You can change the ribbon's name ("TEST") by editing the default system
+  parameter "ribbon.name" (in the menu Settings > Parameters > System
+  Parameters) To hide the ribbon, set this parameter to "False" or delete it.
```

Comment:
Setting all the ribbon parameters to False does not hide the ribbon:
![image](https://user-images.githubusercontent.com/32064796/46356333-d48e7280-c662-11e8-86e5-96f3329d4180.png)

Deleting them results in 
```
Odoo Server Error
Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 653, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 312, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/.repo_requirements/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/.repo_requirements/odoo/odoo/http.py", line 695, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 344, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 337, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 938, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 517, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 955, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 947, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 745, in call_kw
    return _call_kw_model(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 718, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/home/odoo/build/OCA/web/web_environment_ribbon/models/web_environment_ribbon_backend.py", line 32, in get_environment_ribbon
    name = self._prepare_ribbon_name()
  File "/home/odoo/build/OCA/web/web_environment_ribbon/models/web_environment_ribbon_backend.py", line 23, in _prepare_ribbon_name
    return name_tmpl.format(**vals)
AttributeError: 'bool' object has no attribute 'format'
```